app.py

```diff
@@ -2,13 +2,31 @@
 from application import create_app, db
 from application.models.exercise import Exercise
 from application.models.timer import Timer
+from application.models.work_session import WorkSession
+from application.models.rest_session import RestSession
 from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy import MetaData
+from sqlalchemy import MetaData, create_engine
 import os
+from sqlalchemy.orm import sessionmaker
 
 app = create_app("development")
 db = SQLAlchemy(app)
 
+@app.cli.command('db_reset')
```

Comment:
I was trying to figure something like this out for a while! Excellent job!

Reply:
I LOVE this db_reset method!!! Super smart @Chulstro